ui/src/app/utils/runtime-utils.tsx

```diff
@@ -0,0 +1,114 @@
+import {runtimeApi} from 'app/services/swagger-fetch-clients';
```

Comment:
This file was renamed to a `.tsx` file.
The only change was the addition of `withRuntimeState` for class components